synopsys-task/src/synopsys-task/enum/ErrorCodes.ts

```diff
@@ -0,0 +1,34 @@
+export enum ErrorCode {
+  SUCCESSFULLY_COMPLETED = 0,
+  UNDEFINED_ERROR_FROM_BRIDGE = 1,
+  ADAPTER_ERROR = 2,
+  BRIDGE_SHUTDOWN_FAILURE = 3,
+  BRIDGE_BREAK_ENABLED = 8,
+  BRIDGE_INITIALIZATION_FAILED = 9,
+  MISSING_AT_LEAST_ONE_SCAN_TYPE = 101,
```

Comment:
if you can add comment that ado related error code starting here that will be good